### DISCUSSION TITLE
Improvement to stretched mode

### DISCUSSION THREAD

**Discussion by Haxarus** [runelite/runelite#18270] [Ideas]
Can you update the stretched mode plugin so each UI element can be resize to a different size.
For example, inventory scaled to 80%, minimap scaled to 40%.

**Comment by lincolnsemailaddress-afk** [comment]
![CurrentCondition](https://github.com/user-attachments/assets/6d251c69-2e4b-4f83-9ff2-4127b68e107a)
![ProposedChange](https://github.com/user-attachments/assets/3d7b2900-4d44-4ee7-8e8f-de1edd89feb2)

Someone look into this please. 2 attachments showing what stretching does to the overheads and hitsplats depending on scale. Even if just for NPC, Other Player, and self 2D rescaling. Big difference in visibility when moving around especially when playing with a group.
